policyengine_us/parameters/gov/states/ca/tax/income/alternative_minimum_tax/exemption/amt_threshold/lower.yaml

```diff
@@ -0,0 +1,30 @@
+description: California provides a flat exemption amount for filers having Alternative Minimum Taxable Income lower than this amount.
+metadata:
+  label: California lower threshold for exemption amount
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2021 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+    href: https://www.ftb.ca.gov/forms/2021/2021-540-p.pdf#page=1
+  - title: 2022 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+    href: https://www.ftb.ca.gov/forms/2022/2022-540-p.pdf#page=1
+  - title: Cal. Rev. & Tax. Code § 17062 
```

Comment:
add section of the legal code

Reply:
for most of the parameters i can't find specific reference in this legal code: https://casetext.com/statute/california-codes/california-revenue-and-taxation-code/division-2-other-taxes/part-10-personal-income-tax/chapter-21-alternative-minimum-tax/section-17062-alternative-minimum-tax
any other legal code i can loot at?